addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2408,3 +2505,37 @@ we-button:hover .o_we_shape_animated_label > span {
     // also not work to override important inline style... this is a limitation.
     transition: none !important;
 }
+
+// GRID LAYOUT
+we-button.o_grid {
+    min-width: fit-content;
+    padding-left: 3px !important;
+    padding-right: 3px !important;
+}
+
+we-select.o_grid we-toggler {
+    width: fit-content !important;
+}
+
+we-button-group.o_grid {
+    min-width: fit-content !important;
+}
+
+// Background grid.
+.o_background_grid {
+    padding: 0;
+}
+
+.o_cell {
```

Comment:
Not specific enough name, in general for web_editor / editor class we use o_we_XXX naming or maybe oe_XXX